api/swagger.yml

```diff
@@ -487,10 +536,16 @@ paths:
           required: false
           schema:
             $ref: '#/components/schemas/BJJ'
-        - name: accountIndex
+        - name: fromAccountIndex
+          in: query
+          required: false
+          description: Only get transactions sent from a specific account. Incompatible with the queries `tokenId`, `hezEthereumAddress` and `BJJ`.
```

Comment:
Maybe it's good to describe somewhere if we are using the `fromAccountIndex` and the `toAccountIndex` together it's a `OR` condition.